feat(doc): doc updates (#3164)
scaleway/docs-content: dedibox/vps/how-to/reinstall-vps.mdx

---
meta:
  title: How to reinstall a Dedibox VPS
  description: This page explains how to reinstall a Dedibox VPS
content:
  h1: How to reinstall a Dedibox VPS
  paragraph: This page explains how to reinstall a Dedibox VPS
tags: vps dedibox reinstall
dates:
  validation: 2024-05-15
  posted: 2024-05-15
categories:
  - dedibox-vps
---

> You can reinstall your VPS at any time from your Dedibox console. To do so, follow these steps.

Reviewer: ```suggestion
You can reinstall your VPS at any time from your Dedibox console. 
```

<Macro id="requirements" />

- A Dedibox account logged into the [console](https://console.online.net)
- [Created a Dedibox VPS](/dedibox/vps/how-to/create-vps/)

> 1. Open the [Dedibox console](https://console.online.net/) home page and locate **Dedibox VPS** in the top navigation menu. Click it to access your list of virtual machines.

Reviewer: ```suggestion
1. Log in to your [Dedibox console](https://console.online.net/) account and locate **Dedibox VPS** in the top navigation menu. Click it to access the list of your Dedibox VPS.
```

> 2. Select the Dedibox VPS you want to manage and click **Manage** next to it. This action will direct you to the VPS information dashboard.

Reviewer: ```suggestion
2. Select the Dedibox VPS you want to reinstall and click **Manage** next to it. This action will direct you to the VPS information dashboard.
```

    <Message type="tip">

>       You can also directly access the VPS information dashboard by directly clicking

Reviewer 2: ```suggestion
      You can also directly access the VPS information dashboard by clicking
```

    	the VPS Hostname or IP address.
    </Message>

> 3. In the dashboard, locate and click **Reinstall**. This displays the reinstallation wizard.

Reviewer: ```suggestion
3. In the dashboard, locate and click **Reinstall**. The reinstallation wizard displays.
```

> 4. Choose your desired **Operating System (OS)** for the virtual machine and select the OS version.

Reviewer: ```suggestion
4. Choose your desired **Operating System (OS)** for the Dedibox VPS and select the OS version.
```

5. Optionally paste an SSH key that can used for authentication when connecting to the Dedibox VPS using SSH.

> 6. Confirm your selection by clicking **Reinstall** then click **OK** in the pop-up prompt to confirm the action. Your existing preferences will be saved, and the reinstallation process will begin. Reinstalling your Dedibox VPS will take a few minutes. The VPS management page remains unavailable during reinstallation.

Reviewer: ```suggestion
6. Confirm your selection by clicking **Reinstall** then click **OK** in the pop-up prompt to confirm the action. Your existing preferences will be saved, and the reinstallation process will begin. 
```
